packages/harpguru-core/src/utils/index.ts

```diff
@@ -5,8 +5,11 @@ export { getColors } from './get-colors'
 export { determineZoomId } from './determine-zoom-id'
 export { determineMatrixDimensions } from './determine-matrix-dimensions'
 export { determineNextColumnBounds } from './determine-next-column-bounds'
+export { extractHarpFaceMatrix } from './extract-harpfacematrix'
 export { isMatchActiveIds } from './ismatch-activeids'
 export { isMatchColumnBounds } from './ismatch-columnbounds'
+export { isMatchHighOrderTuples } from './ismatch-highordertuples'
```

Comment:
Is this still used? If it's only used for the layoutFacts then it can be removed if we get a bit more type parity between the layout facts and the harpfacematrices.

Reply:
Removed in https://github.com/js-jslog/harpguru/pull/130